task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -55,8 +60,11 @@ def kill(self, task_id):
 
     def stop(self):
         self.execution_framework.stop()
+        log.debug("stopped the framework")
```

Comment:
nit: I think generally single quotes are preferred in general unless your string has single quotes in it

https://confluence.yelpcorp.com/display/ENG/Python+Style+Guidelines#PythonStyleGuidelines-Strings